src/components/20-molecules/datepicker/ui.test.js

```diff
@@ -710,7 +690,7 @@ test('should allow year ranges', async t => {
     return JSON.stringify(domNode[property]);
   });
 
-  await t.setTestSpeed(0.5);
+  await t.setTestSpeed(0.25);
```

Comment:
didn't we say this fiddling with test speed is a sign that this particular test, too, should be rewritten in PlayWright?

Reply:
Yes, this is correct. But i think for the moment it is too complex to migrate (this single test). Since we have special functions for the testcafe tests. I think it would make more sense to migrate all tests for this component in one step.
If you think different, let us discuss about :-)

Reply:
As agreed orally, we'll fix this in a separate issue; thus, for now this is OK here.